src/OpenLoco/src/Windows/Vehicle.cpp

```diff
@@ -1048,6 +1048,15 @@ namespace OpenLoco::Ui::Windows::Vehicle
             return self;
         }
 
+        // ???
```

Comment:
I believe this function was missing. Need to check if it was there originally, and add the address if so.

Reply:
Looks like it wasn't in original. Also we need this for the main tab.